docling_core/experimental/idoctags.py

```diff
@@ -1,199 +1,933 @@
+    | 27 |  | `form_text` | No | No | — | Form text block. |
+    | 28 |  | `hint` | No | No | — | Hint for a fillable field (format/example/description). |
+    | 29 | Grouping Tokens | `section` | No | Yes | `level` | Document section; `level` (N ≥ 1). |
+    | 30 |  | `list` | No | Yes | `ordered` |
+      List container; optional `ordered` in {`true`,`false`} defaults to
+      `false`. |
+    | 31 |  | `group` | No | Yes | `type?` |
+      Generic group; no `location` tokens; associates composite content
+      (e.g., captions/footnotes). |
+    | 32 |  | `floating_group` | No | Yes | `class` in {`table`,`picture`,`form`,`code`} |
+      Floating container that groups a floating component with its caption,
+      footnotes, and metadata; no `location` tokens. |
+    | 33 | Formatting Tokens | `bold` | No | No | — | Bold text. |
+    | 34 |  | `italic` | No | No | — | Italic text. |
+    | 35 |  | `strikethrough` | No | No | — | Strike-through text. |
+    | 36 |  | `superscript` | No | No | — | Superscript text. |
+    | 37 |  | `subscript` | No | No | — | Subscript text. |
+    | 38 |  | `rtl` | No | No | — | Right-to-left text direction. |
+    | 39 |  | `inline` | No | Yes | `class` in {`formula`,`code`,`picture`} |
+      Inline content; if `class="picture"`, may include one of `base64` or
+      `uri`. |
+    | 40 |  | `br` | Yes | No | — | Line break. |
+    | 41 | Structural Tokens (OTSL) | `otsl` | No | No | — | Table structure container. |
+    | 42 |  | `fcel` | Yes | No | — | New cell with content. |
+    | 43 |  | `ecel` | Yes | No | — | New cell without content. |
+    | 44 |  | `ched` | Yes | No | — | Column header cell. |
+    | 45 |  | `rhed` | Yes | No | — | Row header cell. |
+    | 46 |  | `corn` | Yes | No | — | Corner header cell. |
+    | 47 |  | `srow` | Yes | No | — | Section row separator cell. |
+    | 48 |  | `lcel` | Yes | No | — | Merge with left neighbor (horizontal span). |
+    | 49 |  | `ucel` | Yes | No | — | Merge with upper neighbor (vertical span). |
+    | 50 |  | `xcel` | Yes | No | — | Merge with left and upper neighbors (2D span). |
+    | 51 |  | `nl` | Yes | No | — | New line (row separator). |
+    | 52 | Continuation Tokens | `thread` | Yes | Yes | `id` |
+      Continuation marker for split content; reuse same `id` across parts. |
+    | 53 |  | `h_thread` | Yes | Yes | `id` | Horizontal stitching marker for split tables; reuse same `id`. |
+    | 54 | Binary Data Tokens | `base64` | No | No | — | Embedded binary data (base64). |
+    | 55 |  | `uri` | No | No | — | External resource reference. |
+    | 56 | Content Tokens | `marker` | No | No | — | List/form marker content. |
+    | 57 |  | `facets` | No | No | — | Container for application-specific derived properties. |
+    | 58 | Structural Tokens (Form) | `key` | No | No | — | Form item key (child of `form_item`). |
+    | 59 |  | `value` | No | No | — | Form item value (child of `form_item`). |
+    """
```

Comment:
I would remove the table from the code itself.

Later, one can just add a link/reference to the actual standard table.

Reply:
fixed